java2typescript-jackson/src/main/java/java2typescript/jackson/module/visitors/TSJsonFormatVisitorWrapper.java

```diff
@@ -98,7 +98,7 @@ public EnumType parseEnumOrGetFromCache(Module module, JavaType javaType) {
 		if (namedType == null) {
 			EnumType enumType = new EnumType(name);
 			for (Object val : javaType.getRawClass().getEnumConstants()) {
-				enumType.getValues().add(val.toString());
+				enumType.getValues().add(((Enum<?>)val).name());
```

Comment:
@raphaeljolivet, could You comment why toString() was used before instead of name()?

(I'm away from the computer for 2 weeks, so i can't look into it properly.)